### PR TITLE
Added support for inline block comment

### PR DESCRIPTION
**Pull request by fracalo**
fixed an issue with this grammar not supporting block-comments after unterminated property values

### CODE REVIEW

**Comment by 50Wliu** [comment]
Can you please rebase this on top of master?  Thanks!

**Comment by fracalo** [comment]
should be ok now; let me know if something missing.

File: spec/css-spec.coffee [hunk 1]
```diff
@@ -180,6 +180,27 @@ describe 'CSS grammar', ->
       expect(tokens[11]).toEqual value: '*/', scopes: ['source.css', 'meta.at-rule.media.css', 'comment.block.css', 'punctuation.definition.comment.css']
       expect(tokens[12]).toEqual value: ')', scopes: ['source.css', 'meta.at-rule.media.css']
 
+  describe 'inline comments', ->
```
Comment (50Wliu) [inline spec/css-spec.coffee]: These tests can probably just go under the generic `comments` block.

File: spec/css-spec.coffee [hunk 2]
```diff
@@ -180,6 +180,27 @@ describe 'CSS grammar', ->
...
+    it 'on same line', ->
```
Comment (50Wliu) [inline spec/css-spec.coffee]: `it 'tokenizes inline single-line comments`

File: spec/css-spec.coffee [hunk 3]
```diff
@@ -180,6 +180,27 @@ describe 'CSS grammar', ->
...
+      {tokens} = grammar.tokenizeLine 'section {border:4px/*padding:1px*/}'
+
+      expect(tokens[7].value).toBe '/*'
+      expect(tokens[7].scopes).toContain 'comment.block.css'
+      expect(tokens[8]).toEqual value: 'padding:1px', scopes: ['source.css', 'meta.property-list.css', 'meta.property-value.css', 'comment.block.css']
+      expect(tokens[9].value).toBe '*/'
+      expect(tokens[9].scopes).toContain 'comment.block.css'
+
+    it 'on multi-line', ->
```
Comment (50Wliu) [inline spec/css-spec.coffee]: `it 'tokenizes inline multi-line comments`

**Comment by fracalo** [comment]
added a little consistency  (passed to `toEqual`)
I agree with you that maintaining a consistent styleguide is always good thing.

**Comment by 50Wliu** [comment]
Thanks!

**Pull request merged**